mne/tests/test_epochs.py

```diff
@@ -2049,6 +2049,14 @@ def test_concatenate_epochs():
     epochs2.event_id = dict(a=2)
     assert_raises(ValueError, concatenate_epochs, [epochs1, epochs2])
 
+    # check events are shifted, but relative position are equal
+    epochs_list = [epochs.copy() for ii in range(3)]
+    epochs_cat = concatenate_epochs(epochs_list)
+    for ii in range(3):
+        evs = epochs_cat.events[ii * len(epochs):(ii + 1) * len(epochs)]
+        rel_pos = epochs_list[0].events[:, 0] - evs[:, 0]
+        assert_true(sum(rel_pos - rel_pos[0]) == 0)
```

Comment:
This is simply testing that events position in the concatenated epoch are equal to the events position in the original file ( minus a constant offset identical for all events). I fixed a small issue with it, but i don't think this PR require a more advanced test.